CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/Edge.java

```diff
@@ -19,33 +19,33 @@
  *
  * @author algol
  *
- * @param <NodeType> the type of nodes the graph connects to.
+ * @param <Node> the type of nodes this edge connects to.
  */
-public class Edge<NodeType extends NodeBase> {
+public class Edge<Node extends NodeBase> {
 
-    private final NodeType source;
-    private final NodeType target;
+    private final Node source;
+    private final Node target;
```

Comment:
Weren't these changes made in one of your other PRs?

Reply:
Yep - this branches off that and so contains the same changes.